packages/wonder-blocks-tooltip/components/tooltip-bubble.js

```diff
@@ -60,6 +98,8 @@ export default class TooltipBubble extends React.Component<Props> {
                 id={id}
                 role="tooltip"
                 data-placement={placement}
+                onMouseEnter={() => this._handleMouseEnter()}
```

Comment:
you might don't need to wrap this with an arrow function.

This should work: `onMouseEnter={this._handleMouseEnter}`